app/models/translation.rb

```diff
@@ -39,6 +40,22 @@ def delete_draft!
     return :bad_request
   end
 
+  def translated_pages
+    translated_pages = []
+
+    resource.pages.each do |resource_page|
+      custom_pages.each do |custom_page|
+        if resource_page.id == custom_page.page_id
+          translated_pages.push(custom_page)
+        else
+          translated_pages.push(resource_page)
+        end
+      end
+    end
+
+    translated_pages
```

Comment:
Yea, that looks like a simpler version.  What's the difference between `map` and `each?`  I'm not sure which one it is when I click Command+B in RubyMine.